### PR TITLE
Added delete many beneficiaries

### PR DESCRIPTION
**Pull request by PepiPetrov**
Added delete many beneficiaries route.

### CODE REVIEW

File: apps/api/src/beneficiary/beneficiary.service.ts [hunk 1]
```diff
@@ -41,4 +41,10 @@ export class BeneficiaryService {
     if (!result) throw new NotFoundException('sorry id not found')
     return result
   }
+
+  async removeManyBeneficiaries(ids: string[]) {
+    return ids.map(async (x) => {
+      return await this.removeBeneficiary(x)
+    })
+  }
```
Comment (kachar) [inline apps/api/src/beneficiary/beneficiary.service.ts:49]: This code is sending queries to the database in a loop that is not optimal and contradicts the best practices.

A much more leaner approach would be to collect all ids and do a `deleteMany` query with `in: ids` clause.

It will end up as a single query deleting many records at once.

File: apps/api/src/domain/generated/coordinator/dto/connect-coordinator.dto.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
-export class ConnectCoordinatorDto {
-  id?: string
-  personId?: string
-}
+
```
Comment (kachar) [inline apps/api/src/domain/generated/coordinator/dto/connect-coordinator.dto.ts:1]: Please run `yarn format` prior your commits so no other auto generated code is affected

File: migrations/20220201085056_pulling_data/migration.sql [hunk 3]
```diff
@@ -1 +0,0 @@
--- This is an empty migration.
```
Comment (kachar) [inline migrations/20220201085056_pulling_data/migration.sql:1]: This change is not related to the PR, please undo it

File: migrations/20220213130743_/migration.sql [hunk 4]
```diff
@@ -1,8 +0,0 @@
-/*
```
Comment (kachar) [inline migrations/20220213130743_/migration.sql:1]: This change is not related to the PR, please undo it

**Review by kachar — CHANGES_REQUESTED** [review]

File: migrations/20220213130743_/migration.sql (hunk 4 above)
Comment (PepiPetrov) [reply migrations/20220213130743_/migration.sql:1]: Done.

File: migrations/20220201085056_pulling_data/migration.sql (hunk 3 above)
Comment (PepiPetrov) [reply migrations/20220201085056_pulling_data/migration.sql:1]: Done.

File: apps/api/src/domain/generated/coordinator/dto/connect-coordinator.dto.ts (hunk 2 above)
Comment (PepiPetrov) [reply apps/api/src/domain/generated/coordinator/dto/connect-coordinator.dto.ts]: Done.

File: apps/api/src/beneficiary/beneficiary.service.ts (hunk 1 above)
Comment (PepiPetrov) [reply apps/api/src/beneficiary/beneficiary.service.ts]: Done.

File: apps/api/src/beneficiary/beneficiary.controller.ts [hunk 5]
```diff
@@ -54,4 +54,13 @@ export class BeneficiaryController {
   async deleteById(@Param('id') id: string) {
     return await this.beneficiaryService.removeBeneficiary(id)
   }
+
+  @Post('delete-many')
+  @Roles({
+    roles: [RealmViewSupporters.role, ViewSupporters.role],
+    mode: RoleMatchingMode.ANY,
+  })
+  async deleteMany(@Body() data: string[]) {
```
Comment (kachar) [inline apps/api/src/beneficiary/beneficiary.controller.ts:63]: An idea for future improvement is to extract the `data: string[]` in own `DeleteBeneficiaryDto` to make sure validation is applied to the body before it hits the serivce.

File: db/seed/vault.seed.ts [hunk 6]
```diff
@@ -12,26 +12,14 @@ export async function vaultSeed() {
   }
 
   const insert = await prisma.vault.createMany({
-    data: [
-      {
-        name: faker.finance.accountName(),
+    data: [...Array(20).keys()].map(() => {
```
Comment (kachar) [inline db/seed/vault.seed.ts:15]: Nice improvement, thanks

File: migrations/20220213130743_/migration.sql (hunk 4 above)
Comment (kachar) [reply migrations/20220213130743_/migration.sql]: Actually this has not been done so I've done it on your branch

```
$ git rev-list -n 1 HEAD --  migrations/20220201085056_pulling_data/migration.sql
513e223ede1f8ddf9941dc9bda8c3be827111694

$ git checkout 513e223ede1f8ddf9941dc9bda8c3be827111694^ --  migrations/20220201085056_pulling_data/migration.sql migrations/20220213130743_/migration.sql
```

Comment (PepiPetrov) [reply migrations/20220213130743_/migration.sql]: OK.

File: apps/api/src/beneficiary/beneficiary.controller.ts (hunk 5 above)
Comment (PepiPetrov) [reply apps/api/src/beneficiary/beneficiary.controller.ts]: Done.

**Review by kachar — APPROVED** [review]
